### PR TITLE
Replace Twitter logos by X

### PR DESCRIPTION
**Pull request by Klemart3D**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Replace Twitter logo by X
| Type?             | Improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | https://github.com/PrestaShop/PrestaShop/issues/36470
| Sponsor company   | Klemart3D
| How to test?      | After compilation, ps_sharebuttons block must display X logo in place of Twitter: X

### CODE REVIEW

**Review by nicosomb — APPROVED** [review]

**Review by boherm — APPROVED** [review]

**Review by florine2623 — APPROVED** [review]
Thanks @Klemart3D !

LGTM ✅ 


https://github.com/PrestaShop/classic-theme/assets/16019289/6107db2c-717a-474f-8782-114ca86b5d63

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Comment by boherm** [comment]
Thanks @Klemart3D!

**Pull request merged**
